src/components/ListItem/ListItem.test.js

```diff
@@ -1,13 +1,9 @@
 import React from "react";
-import ReactDOM from "react-dom";
 import { ListItem } from "./index.js";
 import { shallow, mount } from "enzyme";
-import { mapDispatchToProps } from "../../containers/App";
```

Comment:
Nice, i think we forgot to take these out when we made  ListItem into a component and CardContainer to a container